sdk/dapp-kit/src/hooks/useSuiClientInfiniteQuery.ts

```diff
@@ -53,9 +57,7 @@ export function useSuiClientInfiniteQuery<T extends keyof SuiRpcPaginatedMethods
 		...options,
 		queryKey: [suiContext.network, method, params, ...queryKey],
 		enabled,
-		queryFn: async () => {
-			return await suiContext.client[method](params as never);
-		},
+		queryFn: () => suiContext.client[method](params as never),
```

Comment:
Actually don't you need to take the page param in here and pass it in subsequent calls?